infra/label_evaluator_step_functions/lambdas/unified_receipt_evaluator.py

```
@@ -1200,6 +1259,14 @@ async def run_geometric() -> tuple[dict, float]:
                                     dynamo_client=dynamo_client,
                                     execution_id=execution_id,
                                 )
+                if chroma_client and hasattr(chroma_client, "close"):
+                    try:
+                        chroma_client.close()
+                    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)